test/lexical/collections/LexicalHeadingFeature/e2e.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, test } from '@playwright/test'
+import { AdminUrlUtil } from 'helpers/adminUrlUtil.js'
+import { reInitializeDB } from 'helpers/reInitializeDB.js'
+import { lexicalHeadingFeatureSlug } from 'lexical/slugs.js'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+import { ensureCompilationIsDone } from '../../../helpers.js'
+import { initPayloadE2ENoConfig } from '../../../helpers/initPayloadE2ENoConfig.js'
+import { TEST_TIMEOUT_LONG } from '../../../playwright.config.js'
+import { LexicalHelpers } from '../utils.js'
+const filename = fileURLToPath(import.meta.url)
+const currentFolder = path.dirname(filename)
+const dirname = path.resolve(currentFolder, '../../')
+
+const { beforeAll, beforeEach, describe } = test
+
+// Unlike the other suites, this one runs in parallel, as they run on the `lexical-fully-featured/create` URL and are "pure" tests
+test.describe.configure({ mode: 'parallel' })
```

Comment:
This option is sadly flaky if there are more than one tests. If you run them in parallel, it will call `reInitializeDB` in parallel, which could conflict.

We'd need either need separate databases in order for this to work reliably, or guarantee that tests don't write to the DB and remove the seeding inside beforeEach

Reply:
> I think this is fine because it doesn't write or delete documents; everything happens in /[collection]/create, without saving anything.

The tests themselves don't write to the db, but `reInitializeDB` does. If that happens in parallel, there can be conflicts. E.g. this happens on postgres when running the lexical fully features test suite:

<img width="2286" height="1486" alt="Screenshot 2025-09-19 at 09 04 11@2x" src="https://github.com/user-attachments/assets/7ec3ec1a-9771-4fde-bc33-9688bc585765" />

We haven't seen those in PR because we forgot to activate running this test suite in PR (already have a PR that addresses that)